### PR TITLE
Expose and airflow chart and image versions

### PR DESCRIPTION
**Pull request by schnie**
This PR re-arranges and adds new configuration to Houston for the default Airflow chart version, as well as the default Image tag and closes https://github.com/astronomer/issues/issues/589.

### CODE REVIEW

**Comment by schnie** [comment]
@andriisoldatenko we can consume this new image tag when we `astro dev init` now.

**Comment by andriisoldatenko** [comment]
@schnie do you mean to change astro-cli according to this new query?

**Comment by schnie** [comment]
Yep this is the API portion of what we are discussing over here https://github.com/astronomer/issues/issues/335

File: src/schema.graphql [hunk 1]
```diff
@@ -140,9 +140,11 @@ type DeploymentConfig {
   astroUnit: AstroUnit
   maxExtraAu: Int
   executors: JSON
-  latestVersion: String
   singleNamespace: Boolean!
   loggingEnabled: Boolean!
+  latestVersion: String @deprecated
+  airflowChartVersion: String!
+  airflowImageTag: String!
```
Comment (ashb) [inline src/schema.graphql]: This one feels slightly out of placed - tag for what image?

Comment (ashb) [reply src/schema.graphql]: Oh sorry, I thought this was on Deployment, not DeploymentConfig

**Comment by schnie** [comment]
This is tied with this change in the airflow helm chart https://github.com/astronomer/helm.astronomer.io/blob/65c1a61e725476e8a69e79e9cb8fd3861916add0/charts/airflow/values.yaml#L59

**Pull request merged**